client/ayon_resolve/plugins/publish/precollect_plates.py

```diff
@@ -0,0 +1,20 @@
+import pyblish
+
+
+class PrecollectPlate(pyblish.api.InstancePlugin):
+    """PreCollect new plates."""
+
+    order = pyblish.api.CollectorOrder - 0.48
+    label = "Precollect Plate"
+    hosts = ["resolve"]
+    families = ["plate"]
+
+    def process(self, instance):
+        """
+        Args:
+            instance (pyblish.Instance): The shot instance to update.
+        """
+        # Temporary disable no-representation failure.
+        # TODO not sure what should happen for the plate.
+        instance.data["folderPath"] = instance.data.pop("hierarchy_path")
+        instance.data["integrate"] = False 
```

Comment:
```suggestion

```

Reply:
@jakubjezek001 Can you explain this suggestion?

Reply:
We also have to include following instance data key for every instance which is having nonexistent hierarchy at the moment of publishing. 

`instance.data["newHierarchyIntegration"] = True`

Reply:
Yep I had the "newHierarchyIntegration" set but I can confirm adding the "clip" family to plate now gives it a representation.